driver/src/test/java/org/neo4j/driver/util/Neo4jRunner.java

```diff
@@ -83,9 +83,10 @@ public Neo4jRunner() throws IOException
         {
             if ( neo4jHome.exists() )
             {
-                System.out.println( "Found an old Neo4j server in: " + neo4jHome.getAbsolutePath() + ". Deleting to " +
-                                    "use a new one." );
-                FileTools.deleteRecursively( neo4jHome );
+                return;
+//                System.out.println( "Found an old Neo4j server in: " + neo4jHome.getAbsolutePath() + ". Deleting to " +
+//                                    "use a new one." );
+//                FileTools.deleteRecursively( neo4jHome );
```

Comment:
If we want to default to use the existing server rather than downloading from the server again, we should make changes to revert this commit https://github.com/neo4j/neo4j-java-driver/commit/b61d475ff9d8192342f5c7039b66e57169d740c0

The reason to change to always download a new server is to avoid using the old server by mistake when running tests locally in IDE

But maybe it might be better to change it back to the old behaviour and call
`FileTools.deleteRecursively( neo4jHome );` inside `private void stopOnExit()`